letsencrypt/client/apache/obj.py

```diff
@@ -58,7 +58,7 @@ class VirtualHost(object):
 
     """
 
-    def __init__(self, filep, path, addrs, ssl, enabled, names=None):
+    def __init__(self, filep, path, addrs, ssl, enabled, names=None):  # pylint: disable=too-many-arguments
```

Comment:
Hah, that was just a singleton instance of the problem, while increasing argument number in pylintrc could lead to more "abuse" :P